sound/soc/sof/intel/Kconfig

```diff
@@ -93,7 +93,9 @@ config SND_SOC_SOF_HDA_COMMON
 	select SND_SOC_ACPI_INTEL_MATCH
 
 config SND_SOC_SOF_HDA
-	tristate
+	tristate "SOF support for HDA Links(HDA/HDMI)"
```

Comment:
Does this have a help section (check patch complains without one).

Reply:
let me add it.